lang/csharp/src/apache/main/Util/TimeMicrosecond.cs

```diff
@@ -25,7 +25,8 @@ namespace Avro.Util
     /// </summary>
     public class TimeMicrosecond : LogicalUnixEpochType<TimeSpan>
     {
-        private static readonly TimeSpan _maxTime = new TimeSpan(23, 59, 59);
+        private static readonly TimeSpan _maxTime = new TimeSpan(24, 00, 00);
```

Comment:
I think a name similar to _exclusiveUpperBound would be more correct now

Reply:
Makes sense.